R/validate_input_data.R

```diff
@@ -1,11 +1,10 @@
-#' Check the input forecast data structure
+#' Check the input forecast data structure and validate it.
 #'
 #' @param forecast_data A data.frame with the predictions that is or can be
 #' coerced to a model_out_tbl format.
 #' @param oracle_output_data A data.frame with the target values.
 #' This data must follow the oracle output format.
-#' @return a model_out_tbl format that contains forecast data and target data
-#' with a single output type.
+#' @return a model_out_tbl format for forecast data
 #'
 #' @import hubUtils
 #' @import dplyr
```

Comment:
same comment as in a previous file. Are examples needed if this function is not exported?